python/pdo/common/keys.py

```diff
@@ -60,6 +57,7 @@ class TransactionKeys(object) :
     """
 
     @classmethod
+    @deprecated
     def read_from_file(cls, file_name, search_path = ['.', './keys']) :
```

Comment:
this doesn't seem necessary. this class may be useful for purposes other than sawtooth. it isn't really deprecated just not explicitly used.

Reply:
Good, that's precisely the doubt that I had.